.github/workflows/ci.yml

```diff
@@ -420,13 +420,11 @@ jobs:
           components: miri
       - uses: Swatinem/rust-cache@v2
       - name: miri
-        # Many of tests in tokio/tests and doctests use #[tokio::test] or
-        # #[tokio::main] that calls epoll_create1 that Miri does not support.
         run: |
-          cargo miri test --features full --lib --no-fail-fast
+          cargo miri test --features full --lib --tests --no-fail-fast
         working-directory: tokio
         env:
-          MIRIFLAGS: -Zmiri-disable-isolation -Zmiri-strict-provenance -Zmiri-retag-fields
+          MIRIFLAGS: -Zmiri-disable-isolation -Zmiri-strict-provenance -Zmiri-retag-fields -Zmiri-tree-borrows
```

Comment:
I also added tree-borrows check because I ran the test suite with it, if this is not desired I can remove them.

Reply:
That is fine. We may want to consider whether it makes sense to run with both?

Reply:
If both means tree borrows and stacked borrows, I think ``-Zmiri-tree-borrows`` will replace stacked borrows with tree borrows rule. On second thought, since tree borrows seem to be more experimental, let me remove this and keep it to stacked borrow.